conf/modules.config

```diff
@@ -1039,11 +1039,11 @@ process{
 
     withName: ".*:ANNOTATION_MERGE:ENSEMBLVEP" {
         // Output file will have format *_snpEff_VEP.ann.vcf
-        ext.prefix       = { "${vcf.minus(".vcf")}_VEP.ann.vcf" }
+        ext.prefix       = { "${vcf.baseName}_VEP.ann.vcf" }
```

Comment:
no minus(".vcf")?

Reply:
baseName already removes the `.vcf`

(running `minus` on `vcf` directly is not working, I suppose because `vcf` is not technically a string)